docs/kqueen.cli_azure_examples.rst

```diff
@@ -0,0 +1,116 @@
+
+In KQueen, you can perform a number of Azure operations using the command-line interface (CLI).
+For example, you can create a Azure account, create a new Kubernetes cluster (AKS), download the kubeconfig and use it to push an application into Kubernetes.
+
+
+To create / configure the Azure to work with KQueen:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Login to your Azure portal:
+
+.. code-block:: bash
+
+   $ az login
+   To sign in, use a web browser to open the page https://microsoft.com/devicelogin and enter the code CBMZ4QPTE to authenticate.
+   [
+     {
+       "cloudName": "AzureCloud",
+       "id": "8xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx2",
+       "isDefault": true,
+       "name": "Pay-As-You-Go",
+       "state": "Enabled",
+       "tenantId": "7xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxd",
+       "user": {
+         "name": "kxxxxxxxxxxxxxxxxxxxxxxxm",
+         "type": "user"
+       }
+     }
+   ]
+
+#. Create new Resource Group
+
+.. code-block:: bash
+
+   $ RESOURCE_GROUP_NAME="kqueen-demo-rg"
+   $ az group create --name "$RESOURCE_GROUP_NAME" --location westeurope
+   {
+     "id": "/subscriptions/8xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxx2/resourceGroups/kqueen-demo",
+     "location": "westeurope",
+     "managedBy": null,
+     "name": "kqueen-demo",
+     "properties": {
+       "provisioningState": "Succeeded"
+     },
+     "tags": null
+   }
+
+#. Create the Service Principal which will have permissions to manage resources in the specified Subscription
+
+.. code-block:: bash
+
+   $ EMAIL=$(az account list | jq -r '.[].user.name')
+   $ SUBSCRIPTION_ID=$(az account list | jq -r ".[] | select (.user.name == \"$EMAIL\").id")
+   $ SECRET="my_password"
+   $ SERVICE_PRINCIPAL_NAME="kqueen-demo-sp"
+   $ az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/$SUBSCRIPTION_ID" --name "$SERVICE_PRINCIPAL_NAME" --password "$SECRET"
+
+#. Get parameters from Azure for KQueen to create provisioner:
+
+.. code-block:: bash
+
+   $ CLIENT_ID="$(az ad sp list --display-name "$SERVICE_PRINCIPAL_NAME" | jq -r '.[].appId')"
+   $ TENANT_ID=$(az ad sp list --display-name "$SERVICE_PRINCIPAL_NAME" | jq -r '.[].additionalProperties.appOwnerTenantId')
+
+
+To create / configure the Azure cluster using KQueen:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+#. Start KQueen and obtain bearer token:
+
+.. code-block:: bash
+
+   $ git clone https://github.com/Mirantis/kqueen.git
+   $ cd kqueen
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml rm -f # Make sure you are starting from scratch
+   $ docker-compose -f docker-compose.yml -f docker-compose.demo.yml up
+   $ TOKEN=$(curl -s -H "Content-Type: application/json" --data '{ "username": "admin", "password": "default" }' -X POST 127.0.0.1:5000/api/v1/auth | jq -r '.access_token')
+   $ echo $TOKEN
+
+#. Create new organization "testorganization" with new user / password "testusername / testpassword":
```

Comment:
If we state in into that ``testorganization`` is used as an example, then we can rename this and other sections. For example:

Create a new organization and add a user